core/src/test/java/io/questdb/test/cutlass/line/tcp/LineTcpConnectionContextTest.java

```diff
@@ -259,8 +259,11 @@ public void testBadLineSyntax1() throws Exception {
                             table + ",location=us-eastcoast temperature=89 1465839830102400200\n" +
                             table + ",location=us-eastcoast temperature=80 a=146583983102400200\n" +
                             table + ",location=us-westcost temperature=82 1465839830102500200\n";
-            handleContextIO();
-            Assert.assertFalse(disconnected);
+            do {
+                handleContextIO();
+                Assert.assertFalse(disconnected);
+            } while (recvBuffer.length() > 0);
```

Comment:
nit: Given it's done so often, maybe it should be a function?

Reply:
done here https://github.com/questdb/questdb/pull/3361/commits/49e7f62b5d63d4c3a159db19ff6f8e493796d800